.github/workflows/build_all.yml

```diff
@@ -56,6 +60,16 @@ jobs:
           else
             orc8r/tools/helm/package.sh --deployment-type all
           fi
+      - name: Launch build and publish script
+        run: |
+          if [ "${GITHUB_REF##*/}" = "master" ] ;then
+            orc8r/tools/helm/package_lf.sh --deployment-type all --version $VERSION  --only-package
+          elif [ "${EVENT_NAME}" = "pull_request" ] ;then
+            mkdir charts
+            orc8r/tools/helm/package_lf.sh --deployment-type all --version $VERSION --only-package
```

Comment:
I do not see much of a difference to the above case of `master` (especially, as the charts folder appears to be unused). So why do we need to make a difference here anyway?

Reply:
I just duplicated the code to have a double push I didn't clean it up. But yes I agree it needs some cleaning